tests/api2/test_system_dataset.py

```diff
@@ -77,3 +77,11 @@ def test_system_dataset_mountpoints():
         assert ds_stats["uid"] == system_dataset_spec["chown_config"]["uid"]
         assert ds_stats["gid"] == system_dataset_spec["chown_config"]["gid"]
         assert ds_stats["mode"] & 0o777 == system_dataset_spec["chown_config"]["mode"]
+
+
+def test_netdata_post_mount_action():
+    # We rely on this to make sure system dataset post mount actions are working as intended
```

Comment:
It should not make a difference, because each time after it runs we expect this to be in place so doing this here will have the same effect.
Should i make the changes?

Reply:
Nope. Just clarifying